docs/i18n-languages/malayalam/how-to-open-a-pull-request.md

```diff
@@ -0,0 +1,43 @@
+<!-- do not translate this -->
+| [Read these guidelines in other languages](/docs/i18n-languages) |
+|-|
+<!-- do not translate this -->
+# ഒരു പുൾ റിക്വസ്റ്റ്(PR) എങ്ങനെ തുറക്കാം
+
+## ഒരു നല്ല പുൾ റിക്വസ്റ്റ് ശീർഷകം എങ്ങനെ തയ്യാറാക്കാം:
+
+ഒരു പുൾ റിക്വസ്റ്റ (PR) തുറക്കുമ്പോൾ, ഇനിപ്പറയുന്ന ഫോർമാറ്റിൽ നിങ്ങളുടെ PR- ന് എന്ത് ശീർഷകം നൽകണമെന്ന് തീരുമാനിക്കാൻ ഇനിപ്പറയുന്ന സ്കോപ്പ് പട്ടിക ഉപയോഗിക്കുക:
+`fix/feat/chore/refactor/docs/perf (scope): PR Title`
+
+ഒരു ഉദാഹരണം `fix(learn): Fixed tests for the do...while loop challenge`.
+
+| Scope | Documentation |
+|---|---|
+| `learn`,`curriculum` | For Pull Requests making changes to the curriculum challenges. |
+| `client` | For Pull Requests making changes to client platform logic or user interface |
+| `guide` | For Pull Requests which make changes to the guide. |
+| `docs` | For Pull Requests making changes to the project's documentation. |
```

Comment:
Should these headings and descriptions be translated @raisedadead ?

Reply:
Yes.